rudder_airflow_provider/hooks/rudderstack.py

```diff
@@ -6,7 +6,7 @@
 from requests.models import Response
 
 STATUS_FINISHED = 'finished'
-STATUS_POLL_INTERVAL = 10
+STATUS_POLL_INTERVAL = 60
```

Comment:
This will increase interval for `/status` endpoint. most of reverse etl syncs take 3-5 mins at minimum to complete. Polling for status 10 seconds is too frequent.

Reply:
60second seems fine. Probably in future we could try to make this adaptive. Say response suggests X records still pending to sync so better to wait 60 and all. Backend API can respond back with some time estimation to wait for next call.